charts/prometheus-node-exporter/values.yaml

```diff
@@ -84,7 +84,9 @@ affinity: {}
 #                 - target-host-name
 
 # Annotations to be added to node exporter pods
-podAnnotations: {}
+podAnnotations:
+  # Fix for very slow GKE cluster upgrades
+  cluster-autoscaler.kubernetes.io/safe-to-evict: "true"
```

Comment:
I don't understand why this PR exists. Are you having trouble passing this map to the existing `annotations` helm value? It appears you can already do this.

Reply:
The purpose of this PR is to provide a new default setting that fixes an issue that all GKE are experiencing (most without knowing why). It is in `values.yaml` to allow everyone to override it if needed.